### PR TITLE
Add deployer deployment

### PR DESCRIPTION
**Pull request by bjorvack**
## Type

- Feature

## Pull request description

Adds deployment using deployer and https://github.com/sumocoders/forkcms_deploy

### CODE REVIEW

**Review by tijsverkoyen — APPROVED** [review]

File: deploy.php [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    run('rm -rf {{release_path}}/.gitattributes');
+    run('rm -rf {{release_path}}/Dockerfile');
+    run('rm -rf {{release_path}}/docker-compose.yml');
+    run('rm -rf {{release_path}}/.scrutinizer.yml');
+    run('rm -rf {{release_path}}/.codecov.yml');
+    run('rm -rf {{release_path}}/php.ini');
+    run('rm -rf {{release_path}}/phpunit.xml.dist');
+    run('rm -rf {{release_path}}/phpstan.neon');
+    run('rm -rf {{release_path}}/UPGRADE***');
+    run('rm -rf {{release_path}}/var/docks');
```
Comment (jonasdekeukelaere) [inline deploy.php:210]: Wat zijn docks?

File: package.json [hunk 2]
```diff
@@ -88,5 +88,13 @@
       "src/Frontend/Cache/",
       "src/Frontend/Core/Js/jquery/jquery.js"
     ]
+  },
+  "volta": {
+    "node": "18.17.1"
```
Comment (jonasdekeukelaere) [inline package.json:93]: Waarom niet de laatste node versie? Dat is al v20 dacht ik?

Comment (bjorvack) [reply package.json]: fixed 491695c970857ba513546729107dfceb18606cf7

File: deploy.php (hunk 1 above)
Comment (bjorvack) [reply deploy.php]: @jonasdekeukelaere fixed in https://github.com/sumocoders/deployer-sumo-forkcms/blob/master/recipe/cleanup.php

**Review by jonasdekeukelaere — APPROVED** [review]

**Pull request merged**